src/accordion/accordion.ts

```diff
@@ -73,8 +74,8 @@ export class NgbPanel {
 }
 
 /**
- * The NgbAccordion directive is a collection of panels.
- * NgbAccordion can assure that only panel can be opened at a time.
+ * The <code>NgbAccordion</code> directive is a collection of panels.
```

Comment:
I would _love_ that we add a markup pass on those docs instead of adding HTML tags.